style.css

```diff
@@ -97,3 +97,127 @@ ul {
   top: 610px;
   right: 2px;
 }
+
+.recent-work {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 120px 24px 120px 24px;
+}
```

Comment:
- [ ] Please make sure that this section which contains all work cards use `display:grid` instead of `display:flex`.

![image](https://user-images.githubusercontent.com/68796232/159141432-bdf39261-25a4-4d21-a616-da301d933014.png)